### DISCUSSION TITLE
How do I clear the cache for all threads

### DISCUSSION THREAD

**Discussion by fredriknilson** [oduvan/django-content-settings#123] [General]
When I change a settings it does not show up in running instances. My application does not use the admin interface, I have tried both using Django shell and the models and directly in the database. The changes works in the shell and other shells but not in the running instance.

Is there a way to clear the cache manually for all threads?

**Comment by oduvan** [comment]
What is your caching configuration?

**Comment by fredriknilson** [comment]
django.core.cache.backends.locmem.LocMemCache

**Reply by oduvan** [reply]
On prod you need to use any caching system. For example memcache or redis.

DCS is using it to signal other threads to update

**Reply by fredriknilson** [reply]
thanks

**Reply by oduvan** [reply]
I have a [ticket](https://github.com/occipital/django-content-settings/issues/114) to make it more obvious when you integrate it
